lib/src/base-styles/_nx-forms.scss

```diff
@@ -8,7 +8,8 @@
 
 // Basic form layout
 .nx-form-group {
-  margin-bottom: $nx-spacing-xs;
+  margin-bottom: $nx-spacing-xl;
+  position: relative;
```

Comment:
Taking into account my other comment, this is unneeded.

Reply:
> Taking into account my other comment, this is unneeded.

https://github.com/sonatype/sonatype-react-shared-components/pull/180/commits/f2226e86881d617085be832ae69ad7df3f02fb30